modules/homePage/component/v2/FourthJumbotron/FourthJumbotron.tsx

```diff
@@ -7,11 +7,11 @@ export default function FourthJumbotron() {
             <h1 className="mb-6 text-center text-[64px] font-extrabold leading-[72px] tracking-[-0.02em] text-dark-neutral-primary md:text-left lg:text-[148px]	lg:leading-[136px]">
                 Dive Deeper <br /> Into Risedle
             </h1>
-            <h3 className="paragraph-xl mb-8 text-dark-neutral-soft">
+            <h3 className="paragraph-xl mb-20 text-dark-neutral-soft">
                 Read our docs, join our Discord, or jump into our <br />
                 Twitter account to get more learning resources!
             </h3>
-            <div className="grid gap-8 md:grid-cols-2 lg:grid-cols-3">
+            <div className="grid gap-8 px-6 md:grid-cols-2 lg:grid-cols-3">
```

Comment:
1. I think the `px-6` is not necessary
2. The gap should've been 24px, which is `gap-6`
3. The breakpoint should've been sm, so `sm:grid-cols-2` (also change the one in twitter section to `sm:col-span-2`)

Reply:
i used `px-6` because i need more padding based on mas ayik review

Reply:
hmmm which one was it? Because if we use `px-6`, these section will not be inline with the sections above
- With `px-6`
![image](https://user-images.githubusercontent.com/81855912/174075046-465b1124-5dba-483f-a05d-fe8ea12c8e20.png)

- Provided in figma:
![image](https://user-images.githubusercontent.com/81855912/174075293-fecb8763-0bbd-41bc-9c30-77629a9935f6.png)